gcs/electron/main.ts

```diff
@@ -81,6 +82,10 @@ app.on('window-all-closed', () => {
     app.quit()
     win = null
   }
+
+  console.log('Killing backend')
+  // kill any processes with the name "fgcs_backend.exe"
+  spawn('taskkill /f /im fgcs_backend.exe', {shell:true})
```

Comment:
Is this just windows or would this work on other platforms?

Reply:
Just windows